stories/core/ColorPicker.stories.tsx

```diff
@@ -0,0 +1,132 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React, { useState } from 'react';
```

Comment:
Very first item in the style guide:

https://github.com/iTwin/iTwinUI-react/blob/fc2b477bdf2cfc77328c358c9e76aa86d7be7007/STYLEGUIDE.md#L11-L12

Also you probably want to import `useState` from storybook instead.

Reply:
Removed